src/Maker/MakeForm.php

```diff
@@ -38,19 +54,62 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
         $command
             ->setDescription('Creates a new form class')
             ->addArgument('name', InputArgument::OPTIONAL, sprintf('The name of the form class (e.g. <fg=yellow>%sType</>)', Str::asClassName(Str::getRandomTerm())))
+            ->addOption('entity', null, InputOption::VALUE_NONE, 'Generate form class from existing entity')
             ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeForm.txt'))
         ;
     }
 
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        $entityFormGeneration = $input->getOption('entity');
```

Comment:
Hmm. This isn't quite what I had in mind, but we can talk about it :). What about this?

> Enter the class or entity name that the new form will be bound to (empty for none):

... and maybe that's it :). If they do not enter anything, then we can ask them for a form class name. Otherwise, we will select a name for them. We can auto-complete this field with entity names, but they should be allowed to enter *any* class name.

Also, I don't think we need the `entity` option. Let's *assume* that the `name` argument (if they pass it in the command) is the "class name" that they want the form bound to. In other words, we'll change the way this command words. It's not really a very useful command *anyways* if you're not binding it to a class.

Reply:
I have just one question is this interaction with user useful? Lets do simple generation 
```
if (class_exists($entity_class_name)): 
  goto: generate_entity_form; 
else:
  goto: generate_simple_form; 
endif;
```
As I think interaction with user may allow us only generate several forms for one entity. If we need this, lets do 2 questions
First:
> The name of the form class (e.g. GentleElephantType):

and Second:
> Enter the class or entity name that the new form will be bound to (empty for none):

And if user passes name as argument than will assume that the inputs are equal.

Reply:
Hmm, ok: let’s do your second option: ask the form class and then ask for the entity class. It’s simple and makes sense

Reply:
@weaverryan how do you think what way is better
- Question with autocomplete
```bash
 Enter the class or entity name that the new form will be bound to (empty for none):
 > SourFood
```
or
- Choice question
```bash
 Enter the class or entity name that the new form will be bound to [none]:
  [0] none
  [1] SourFood
 >
```